stdlib/src/builtin/object.mojo

```diff
@@ -577,6 +577,18 @@ struct _ObjectImpl(CollectionElement, Stringable):
         else:
             lhs = lhs.convert_int_to_float()
 
+    @staticmethod
+    fn coerce_integral_type(inout lhs: _ObjectImpl, inout rhs: _ObjectImpl):
```

Comment:
**Suggestion** Consider making this private (prefix with `_`).  Then, doc-gen won't generate public API docs for this as users shouldn't call this directly.

Reply:
@JoeLoser  I implement this function inside the `_ObjectImpl` struct which I believe is private in nature right?

Reply:
Oops, you're right here - sorry I missed that! 👍